### PR TITLE
Add first in line prompt

### LINKED ISSUE #63: Prompt buyer to message seller when they are front of the line

### LINKED ISSUE #69: Leaving queue notifies buyer that someone has left the queue
Current functionality:
[BUYER]: Leave queue
[BOT]: You have successfully been removed from the queue.
[BOT]: Someone from one of the listings you're watching has left the queue.
[BOT]: You're first in line for <title>.

Expected functionality:
[BUYER]: Leave queue
[BOT]: You have been successfully removed from the queue.

### PR DESCRIPTION
**Pull request by nandanmen**
- fixes #69 
- Prompt user to message seller when they become first in line
- Add checks to ensure they don't get asked again (when a user gets added to the queue for example)
  - Current way is very hacky, ideally we change the queue data structure in firebase

### CODE REVIEW

**Comment by andrewzulaybar** [comment]
@narendrasss Do you think this is related to the issue I just created? #69

**Comment by narendrasss** [comment]
@andrewzulaybar Yeah I literally just fixed it when you put up the issue. I was making changes with the flow to make sure these messages don't interefere.

File: src/db/listings.js [hunk 1]
```diff
@@ -79,14 +82,38 @@ function setSellerPrice(listingId, price) {
   return db.ref(`listings/${listingId}/price`).set(price);
 }
 
-function setQueue(listingId, hasQueue) {
-  return db.ref(`listings/${listingId}/has_queue`).set(hasQueue);
+async function createQueue(listingId) {
+  const listingSnapshot = await db.ref(`listings/${listingId}`).once("value");
+  listingSnapshot.child("has_queue").ref.set(true);
```
Comment (andrewzulaybar) [inline src/db/listings.js]: Is it `.ref()`?

File: src/db/listings.js [hunk 2]
```diff
@@ -79,14 +82,38 @@ function setSellerPrice(listingId, price) {
...
+        prevMessaged[listingId] = firstInLine;
+        send
+          .text(
+            user,
+            `You're now first in line for ${listingSnapshot.val().title}!`
+          )
+          .then(() =>
+            send.text(
+              user,
+              "Please make sure to message the seller within the next 48 hours to keep your place in line. You can do that any time by typing in '\\message seller'."
```
Comment (andrewzulaybar) [inline src/db/listings.js]: Oh I added something like this in `copy.json`. Can you add it in there so it also sends this message for the timeouts?

**Review by andrewzulaybar — CHANGES_REQUESTED** [review]
Just one change!

**Review by andrewzulaybar — APPROVED** [review]

**Pull request merged**